sonar-plugin-api/src/main/java/org/sonar/api/utils/log/ConsoleLogger.java

```diff
@@ -135,6 +135,12 @@ protected void doWarn(String msg) {
   }
 
   @Override
+  void doWarn(String msg, Throwable thrown) {
+    doWarn(msg);
+    thrown.printStackTrace();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1148)